servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingStreamingToStreamingService.java

```diff
@@ -197,11 +202,61 @@ public void close(final Throwable cause) throws IOException {
 
         @Override
         public HttpHeaders trailers() {
```

Comment:
A "getter" method that also creates seems off to me. IDK if there is a better naming scheme where `trailers0` is basically just `trailers` and `trailers` is `getOrCreateTrailers`.

Reply:
It's "lazy get" 😄 
We use the same trick in `AbstractHttpMetaData.context()`. When API is written in stone but we need to defer action. 
From users point of view it will be non-visible change.